test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs

```diff
@@ -654,17 +657,19 @@ private void BuildProjectJson(string projectDirectory)
 
             var result = new BuildPJCommand()
                 .WithCapturedOutput()
+                .WithForwardingToConsole()
                 .Execute(projectFile);
 
             result.Should().Pass();
         }
 
         private void MigrateProject(params string[] migrateArgs)
         {
-            var result =
-                MigrateCommand.Run(migrateArgs);
-
-            result.Should().Be(0);
+            new TestCommand("dotnet")
+                    .WithForwardingToConsole()
+                    .Execute($"migrate {string.Join(" ", migrateArgs)}")
+                    .Should()
```

Comment:
I like to keep each assertion clause on one line. Putting `.Should` and `.Pass` on two lines reduces readability as additional clauses are added e.g.:

```
.Should()
.Pass()
.And
.HaveFiles()
```

vs. 

```
.Should().Pass()
.And.HaveFiles()
```

Thoughts?